coderunners/compilers.py

```diff
@@ -54,3 +56,26 @@ def compile(self, submission_path: Path):
         binary_path.unlink(missing_ok=True)
         executable_path = f'{self.language_standard} {submission_path}'
         return executable_path, compile_res
+
+@dataclass
+class CSharpCompiler(Compiler):
+    language_standard: str
+    supported_standards = {'c#'}
+
+    dotnet_path = Path('/root/.dotnet/dotnet')
+    project_file_path = Path('/tmp/program/program.csproj')
```

Comment:
Wouldn't it be better to keep project_dir instead of accessing `self.project_file_path.parent` in the function?